airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/DbConverter.java

```diff
@@ -193,6 +196,11 @@ public static ActorCatalog buildActorCatalog(final Record record) {
         .withCatalogHash(record.get(ACTOR_CATALOG.CATALOG_HASH));
   }
 
+  public static ActorCatalogFetchEvent buildActorCatalogFetchEvent(final Record record) {
```

Comment:
We need to add all the fields present in the record here only the actor catalog id is set here even if all the other fields are provided.

Reply:
We are also missing the created_at column and updated_at column in the model.